config/config.sample.yaml

```diff
@@ -1,17 +1,40 @@
-# Port for incoming slack requests from webhooks and event API messages.
-slack_hook_port: 9898
-# Prefix of incoming requests to strip when handing. This is NOT the bind host
-  #NOTE: Unlike most of the other urls, this one cannot use localhost,
-  #as this one must be publicly visible to the slack API
-inbound_uri_prefix: "https://my.server.here:9898/"
+homeserver: # Required
+  # The domain name of your homeserver
+  server_name: my.server.here
+  # The URL which the bridge server can reach the homeserver on.
+  # You can use localhost so long as the bridge and homeserver are
+  # hosted on the same machine
+  # 8008 is assumed to be the port the homeserver listens on
```

Comment:
I could see this potentially confusing somebody, putting `8008` at the end of `https://my.domain.com` or something.

I would probably just remove this line.